### ISSUE TITLE
Error when installing Orange via anaconda-navigator

### ISSUE THREAD

**Issue by brylie** [ContinuumIO/anaconda-issues#1113]
When trying to install Orange from the anaconda-navigator, I get the following error:

```py
OS error: failed to link (src='/home/brylie/anaconda3/pkgs/qt-4.8.7-4/mkspecs/default', dst='/home/brylie/anaconda3/mkspecs/default', type=3, error=FileExistsError(17, 'File exists'))
```

**Comment by brylie** [comment]
Note, this may be a remnant of issues I was experiencing managing versions of Qt (related #1107).

**Comment by brylie** [comment]
I tried `rm -rf ~/anaconda3/pkgs/qt-4.*`, but got the same error.

**Comment by brylie** [comment]
I opened a [support request on the Orange 3 issue tracker](https://github.com/biolab/orange3/issues/1613). It is unclear to me where the problem is, so I am following up on both project trackers.

**Comment by csoja** [comment]
@brylie thank you for submitting the issue.  We do not have a Qt 5.x compatible version of Orange2 or Orange3.  As a result, Orange is incompatible with the latest version of Navigator and Spyder.  You can create a new environment just for Orange - that does not have the latest Navigator or Spyder in it, and that should continue to work fine for you.  Do you see this same error when you install Orange into a new environment?

**Comment by brylie** [comment]
@csoja thanks for the response.

For what it's worth, there are now two issues on the Qtpy repository related to getting Orange 3 to work with Qt 5:
- https://github.com/spyder-ide/qtpy/issues/76
- https://github.com/spyder-ide/qtpy/issues/77

**Comment by ccordoba12** [comment]
We're discussing how to solve spyder-ide/qtpy#77.

I'm afraid spyder-ide/qtpy#76 needs to be addressed by the Orange guys (i.e. by changing their calls to `pyqtSignal` to `Signal` instead :-).

**Comment by ijstokes** [comment]
I think I'm having this problem as well, but I'm a bit confused because if the packages aren't available/compatible why does i) Navigator show up `orange3` as an installable package; and ii) actually attempt to install it when I click on it? (though it does appear to get stuck in the "*Linking Qt*" stage of the package install).

**Comment by brylie** [comment]
@ijstokes yes, that does appear to be somewhat of an usability issue.

**Comment by csoja** [comment]
We removed the Orange app from repo.continuum.io - so I'm surprised it is still available on the home screen of Navigator.  We will have to investigate that.  orange is still available on the environments tab (and on repo.continuum.io) - because if you create a new environment, you can install Orange into that new environment and it will work.  

We are investigating how to improve the usability.  Not all packages are compatible with each other, and it is only after conda's solver gets to work when you try to install/update a particular package that you may find out it is not compatible with your current environment.  Plus, as soon as you install one package, then that may change what other packages are now compatible with your environment.

**Comment by brylie** [comment]
> We removed the Orange app from repo.continuum.io

@csoja yesterday, I noticed that Orange3 is no longer visible in the anaconda-navigator home screen.

> you can create a new environment just for Orange

I created a new environment in anaconda-navigator, and installed Orange3 from the package manager. How do I launch Orange3 from its environment?

**Comment by csoja** [comment]
On your Orange environment, click the triangle “Go” button, select Terminal, and in the terminal window that opens type “orange-canvas”. Orange opens up in its own environment, separate from Navigator.

**Comment by jsolderitsch** [comment]
Orange canvas does not work on Mac OS X when spawned from a terminal. There a lot of null pixmap errors. The Orange icons are not decorated like they should be, the tutorial selection images are missing/garbled. I know this issue is closed but this is a pretty serious usability regression.

**Comment by goanpeca** [comment]
@jsolderitsch this issue never had anything to do with what you report. Please open a new issue.

**Comment by jsolderitsch** [comment]
Done -- new issue recorded.
